eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/jdbc/statements/AppendEventsStatementBuilder.java

```diff
@@ -48,5 +48,5 @@
      * @throws SQLException when an exception occurs while creating the prepared statement.
```

Comment:
I expect a `@param timestampWriter` to be added to the javadoc.

Reply:
Added.